src/com/gfredericks/test/chuck/cljs_test.cljc

```diff
@@ -0,0 +1,54 @@
+(ns com.gfredericks.test.chuck.cljs-test
```

Comment:
I'm a bit iffy on the namespace name. test.check will be changing its namespace from `clojure.test.check.cljs-test` to `clojure.test.check.clojure-test` soon I think, so it might make sense for us to do the same thing.